.docs/README.md

```diff
@@ -312,50 +288,43 @@ $payment = $service->restorePayment([
 ]);
 ```
 
-Na objektu platby lze zavolat dvě kontrolní metody: `isFraud()` a `isPaid()`.
-První nás informuje, jestli je platba pravá, respektive nejedná-li se
-o podvrh (interně se zde kontroluje ona čtveřice parametrů předaných z platební
-brány).
+Two control methods can be called on a payment object: `isFraud ()` and `isPaid ()`. The first informs us whether the payment is genuine or not for fraud (internally, the four parameters passed from the payment system are checked by payment gate).
 
-Druhá `isPaid()` pak vrátí `TRUE`, pokud je platba skutečně zaplacena. Pokud
-ano, proces je u konce, můžeme si poznačit, že objednávka je zaplacena a poslat
-třeba zákazníkovi email.
+The second `isPaid ()` then returns `TRUE` if the payment is actually paid. If yes, the process is over, we can mark that the order is paid and send for example, an email to the customer.
```

Comment:
The second `isPaid()` then returns `TRUE` if the payment is actually paid. If yes, the process is over, we can mark that the order is paid and send for example, an email to the customer.